doc/source/examining/loading_data.rst

```diff
@@ -2112,6 +2112,78 @@ For example, to add support for a longint field named
 ``my_custom_field``, one would add ``('my_custom_field', 'l')`` to ``assoc``.
 
 
+Particle ages and formation times
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For cosmological simulations that include star particles, RAMSES stores particle
+formation times as conformal times. To access the formation time field data in
+conformal units use the ``"conformal_birth_time"`` field. This will return
+the formation times of particles in the simulation in conformal units as a
+dimensionless array. To access the formation time in physical units, use the
+``"particle_birth_time"`` field. Finally, to access the ages of star
+particles in your simulation, use the ``"star_age"`` field. Note that this field
+is defined for all particle types but will only make sense for star particles.
+
+For non-cosmological simulations, the particle ages are already in physical
+units on disk, so there will not be a ``"conformal_birth_time"`` field. To
+access the formation time in physical units, use the ``"particle_birth_time"``
+field.
+
+For all simulations, the time is equal to zero at the beginning of the
+simulation. That means there can be situations where the particle formation time
+might be negative. In such cases, the simulation included particles with a
+finite age in the initial conditions. This can happen in idealized isolated
+galaxy simulations, where star particles are included in the initial conditions.
+
+To help clarify the above discussion, the following table describes the meaning
+of the various particle formation time and age fields:
+
++------------------+----------------------------+-------------------------------+
+| Simulation type  | Field name                 | Description                   |
+|==================|============================+===============================+
+| cosmological     | `conformal_birth_time`     | Formation time in conformal   |
+|                  |                            | units (dimensionless)         |
++------------------+----------------------------+-------------------------------+
+| any              | `particle_formation_time`  | The time relative to the      |
+|                  |                            | beginning of the simulation   |
+|                  |                            | when the particle was formed. |
+|                  |                            | Positive for particles formed |
+|                  |                            | during the simulation and     |
+|                  |                            | negative for particles of     |
+|                  |                            | finite age in the initial     |
+|                  |                            | conditions.                   |
++------------------+----------------------------+-------------------------------+
+| any              | `star_age`                 | Age of the particle.          |
```

Comment:
I think it is also valid for particles which were already included at the beginning of the simulation, and not only for particles which formed "dynamically"